NEWS.md

```diff
@@ -43,6 +43,7 @@ Standard library changes
 * `hasmethod` can now check for matching keyword argument names ([#30712]).
 * `startswith` and `endswith` now accept a `Regex` for the second argument ([#29790]).
 * `retry` supports arbitrary callable objects ([#30382]).
+* `filter` now supports `SkipMissing`-wrapped arrays ([#31233]).
```

Comment:
Ah RIP, I tried to guess what PR number this would be and I was wrong. I'll fix it after CI has gone through.